CHANGELOG.md

```diff
@@ -21,6 +21,11 @@
   function parameter can be replaced with an opaque `some` type.  
   [SimplyDanny](https://github.com/SimplyDanny)
 
+* Add `excluded` configuration option to `unneeded_override` to opt out checking methods with a given name.
+  For example, this helps avoid a conflict with `balanced_xctest_lifecycle` where one of `setUp/tearDown`
+  is empty but the other is not.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.
```suggestion
  is empty but the other is not.  
```